docs/README.md

```diff
@@ -0,0 +1,66 @@
+# Onsite-demo documentation
+
+## Description
+
+- [Used features and project decisions](#features)
+- Brief of widget types [here](./widgets-brief.md).
+- [Brief of the recommendation documentation, description of api response](#doc-rec)
+- [General flow of widgets implementation](#flow)
+- Description of each widget type, it's behaviors and singularities [here](./widgets-description).
+- [Code tips and tricks, general and for specific widgets](#tips)
+
+## <a name="features">Used features and project decisions</a>
+
+### List of features
+
+- `jQuery-3.3.1`: https://jquery.com/
+- `EJS`: https://ejs.co/
+- `ESLint`: https://eslint.org/
+- `Webpack`: https://webpack.js.org/
+- `@linx-impulse/commons-js`: https://github.com/chaordic/commons-js/
+- `Owl Carousel 2`: https://owlcarousel2.github.io/OwlCarousel2/
+- `Slick carousel`: http://kenwheeler.github.io/slick/
+- `Onsite-SDK`: https://github.com/chaordic/engage-onsite-sdk-js
+
+### Project decisions and some explanations
+
+- `@linx-impulse/commons-js`: reusing some util functions from this repo.
+- `Onsite-SDK`: SDK interface to use api.
+- `Owl Carousel 2`: plugin used to build the horizontal carousels.
+- `Slick carousel`: plugin used to build the vertical carousels.
+- The project has the components structures for a clear and easy framework migration (such as React, Vue, Angular, etc).
+- The file `utils.js` has some very specific functions, explained on [Code tips and tricks](./tips.md).
+
+## Brief of widget types [here](./widgets-brief.md)
+
+## <a name="doc-rec">Brief of the recommendation doc, description of api response</a>
```

Comment:
esse `a` tá errado?

Reply:
To usando ele pra ser a ancora do markdown

Reply:
talvez